recipe/conda_build_config.yaml

```diff
@@ -1,2 +0,0 @@
-channel_targets:
-  - conda-forge rc_iris
```

Comment:
@lbdreyer As this is a pre-release, don't we need to populate the `rc_iris` channel instead of `main`?